hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestGetCommittedBlockLengthAndPutKey.java

```diff
@@ -110,12 +115,21 @@ public void tesGetCommittedBlockLength() throws Exception {
         ContainerTestHelper
             .getPutBlockRequest(pipeline, writeChunkRequest.getWriteChunk());
     client.sendCommand(putKeyRequest);
-    response = ContainerProtocolCalls
-        .getCommittedBlockLength(client, blockID, null);
+    GenericTestUtils.waitFor(() -> {
+      boolean success = true;
+      try {
+        response[0] = ContainerProtocolCalls
+            .getCommittedBlockLength(client, blockID, null);
+      } catch (IOException e) {
+        success = false;
+        LOG.error("Ignore the exception till wait: {}", e.getMessage());
+      }
+      return success;
+    }, 1000, 5000);
```

Comment:
this would be 5 tries I believe, did you try how much time does it take, if it is fast you can explore reducing 1000 to a lower number

Reply:
What i observed was not very long time, it appears to be in millisec. so good idea to change to lower down the wait check value. I changed now to 500ms.